lib/modules/nightMode.js

```diff
@@ -144,15 +153,26 @@ module.always = () => {
 module.go = () => {
 	handleAutomaticNightMode();
 
-	if (module.options.nightSwitch.value) {
+	if (module.options.nightSwitch.value && className() === 'res-nightmode') {
 		toggle.addMenuItem(i18n('nightModeToggleTitle'), '☽', '☀');
 	}
 };
 
 export function isNightModeOn(): boolean {
 	if (!Modules.isRunning(module)) return false;
+
+	const userCookie = getUserCookie();
+
 	// `toggle` contains the current nightMode state
-	return toggle ? toggle.enabled : module.options.nightModeOn.value;
+	// `userCookie` contains the decoded "USER" cookie data
+	return (toggle ? toggle.enabled : module.options.nightModeOn.value) || (userCookie.prefs.nightmode && className() === 'res-d2x-nightmode');
+}
+
+export function getUserCookie(): RedditUserCookie {
```

Comment:
There might be a better way to do this via the jsapi, @andytuba?

Reply:
Jsapi doesn't have the cookie, but that's a good idea. I'll make a note for next week.

Reply:
What is the "jsapi"? Where is it located? I cannot find anything online related to this term

Reply:
https://www.reddit.com/r/redesign/wiki/jsapi

Reply:
I'd like to come back to "get redesign nightmode pref" later, but this should hold for now.